src/app/editor/editor.component.css

```diff
@@ -68,6 +68,10 @@ sketch-container {
   position: absolute;
 }
 
+.sidebar {
+  min-width: 200px;
```

Comment:
why 200px?

Reply:
Originally the sidebar size was 200px. Without this minimum the sidebar would break up.
![screencast 2018-10-31 17-50-25](https://user-images.githubusercontent.com/22115108/47804691-bbe2ac80-dd35-11e8-9680-5f718e06372e.gif)

Reply:
With i ensure that the sidebar doesn't break.
![screencast 2018-10-31 17-53-15](https://user-images.githubusercontent.com/22115108/47804783-fb10fd80-dd35-11e8-8f41-6a73a00a6ee7.gif)